doc/guides/writing-tests.md

```diff
@@ -30,23 +30,25 @@ const common = require('../common');                                   // 2
 // in the http header.                                                 // 5
 
 const assert = require('assert');                                      // 7
-const http = require('http');                                          // 8
-
-const server = http.createServer(common.mustCall((req, res) => {       // 10
-  res.end('ok');                                                       // 11
-}));                                                                   // 12
-server.listen(0, () => {                                               // 13
-  http.get({                                                           // 14
-    port: server.address().port,                                       // 15
-    headers: { 'Test': 'Düsseldorf' }                                  // 16
-  }, common.mustCall((res) => {                                        // 17
-    assert.strictEqual(res.statusCode, 200);                           // 18
-    server.close();                                                    // 19
-  }));                                                                 // 20
-});                                                                    // 21
+const fixtures = require('../common/fixtures');                        // 8
```

Comment:
can we at least move this to _either_ above or below the core module imports?

Reply:
Maybe we should just put it under the `require('common')` line then, so all the common modules are in the same place.